engine/src/main/java/org/terasology/rendering/dag/stateChanges/SetRenderBufferMask.java

```diff
@@ -24,7 +24,10 @@
 import org.terasology.rendering.dag.StateChange;
 
 /**
- * TODO: Add javadocs
+ * This StateChange generates the tasks that set and reset render masks.
+ *
+ * The rendering pipeline can either write to the Color buffer, the DepthStencil buffer
+ * or the Light buffer, all of which can be toggled independently of each other.
```

Comment:
Ok, two concepts have to emerge here:
1. the render mask is part of the state of an FBO - not the opengl state.
2. this particular StateChange really only works on one of the read/write only FBOs - it doesn't work with any other FBO.